testcases/HelloWorld.py

```diff
@@ -18,17 +18,26 @@
 # permissions and limitations under the License.
 
 import unittest
+import logging
 
 import OpTestConfiguration
+import OpTestLogger
 from common.OpTestUtil import OpTestUtil
 from common.OpTestSystem import OpSystemState
 
+module_logger = logging.getLogger(OpTestLogger.optest_logger_glob.parent_logger+'.{}'.format(__name__))
+
 class HelloWorld(unittest.TestCase):
     def setUp(self):
         conf = OpTestConfiguration.conf
+        self.my_logger = logging.getLogger(OpTestLogger.optest_logger_glob.parent_logger+'.{}'.format(__name__))
```

Comment:
Do we need to have a separate loggers for both setUp and runTest individually or this is just a example usecase?
Wondering the above global logger(`module_logger`) for this module is sufficient for entire module?

Reply:
Kept the module_logger level (it was just for example).